src/pages/profile/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { NextPage } from "next";
+import Page from "src/components/Page";
+import ProfileContent from "src/pageContent/Profile";
+
+const Profile: NextPage = () => {
+  return (
+    <Page
+      bgImage="varrene"
```

Comment:
Hem what does this line do? Right now putting `bgImage="varrene"` won't give you any background image. What should you do in order to add a background image?